docs/source/raft_bft_migration.md

```diff
@@ -0,0 +1,152 @@
+# Migrating from Raft to BFT
+
+Nodes at v3.0.0 or higher allow for users who want to transition channels from using Raft-based ordering services to BFT-based ordering services.
+This can be accomplished through a series of configuration update transactions on each channel in the network.
+
+This tutorial will describe the migration process at a high level, calling out specific details where necessary.
+
+
+## Assumptions and considerations
+Before attempting migration, take the following into account:
+
+1. This process is solely for migration from Raft to BFT. Migrating between any other orderer consensus types is not supported.
+2. Migration is one way. Once the ordering service is migrated to BFT, and starts committing transactions, it is not possible to go back to Raft.
+3. Because the ordering nodes must go down and be brought back up, downtime must be allowed during the migration.
+4. Recovering from a botched migration is possible only if a backup is taken at the point in migration prescribed later in this document. If you do not take a backup, and migration fails, you will not be able to recover your previous state.
+5. All channels must be migrated during the same maintenance window. It is not possible to migrate only some channels before resuming operations.
+6. At the end of the migration process, every channel will have the same consenter set of BFT nodes.
+7. Migration is done in place, utilizing the existing ledgers for the deployed ordering nodes. Addition or removal of orderers should be performed after the migration. 
+
+
+## Migration flow
```

Comment:
Do we still need to upgrade the OSN binary now for the consensus type change?

Reply:
Yes. To migrate, an upgrade from version 2.x to version 3.0.0 should be done.
It is mentioned at the beginning of the document that nodes at v3.0.0 or higher allow the transition to a BFT ordering service, but I will clarify that an upgrade is needed.